operators/advanced_network/README.md

```diff
@@ -162,6 +162,10 @@ bound to a non-isolated core. Should differ from isolated cores in queues below.
   - type: `string`
 - **`log_level`**: Backend log level. default: `warn`. Other: `trace` , `debug`, `info`, `error`, `critical`, `off`
   - type: `string`
+- **`tx_meta_buffers`**: Metadata buffers for transmit. One buffer is used for each burst of packets (default: 4096)
+  - type: `integer`
+- **`rx_meta_buffers`**: Metadata buffers for receive. One buffer is used for each burst of packets (default: 4096)
+  - type: `integer`
```

Comment:
so these are shared for all ports and queues?

Reply:
yes